src/cmake/externalpackages.cmake

```diff
@@ -198,25 +202,29 @@ checked_find_package (TBB 2017
                       PREFER_CONFIG)
 
 checked_find_package (DCMTK VERSION_MIN 3.6.1)  # For DICOM images
-checked_find_package (FFmpeg VERSION_MIN 3.0)
+checked_find_package (ffmpeg
+                      COMPONENTS avcodec avformat avutil swscale
+                      VERSION_MIN 3.0)
 checked_find_package (GIF
                       VERSION_MIN 4
                       RECOMMEND_MIN 5.0
                       RECOMMEND_MIN_REASON "for stability and thread safety")
 
 # For HEIF/HEIC/AVIF formats
-checked_find_package (Libheif VERSION_MIN 1.3
+checked_find_package (libheif VERSION_MIN 1.3
                       RECOMMEND_MIN 1.7
                       RECOMMEND_MIN_REASON "for AVIF support")
 if (APPLE AND LIBHEIF_VERSION VERSION_GREATER_EQUAL 1.10 AND LIBHEIF_VERSION VERSION_LESS 1.11)
     message (WARNING "Libheif 1.10 on Apple is known to be broken, disabling libheif support")
-    set (Libheif_FOUND 0)
+    set (libheif_FOUND 0)
 endif ()
 
-checked_find_package (LibRaw
+checked_find_package (libraw
```

Comment:
This change is gonna make it not use our FindLibRaw.cmake module. Does libraw have an exported cmake config? If so, when did it appear and does that force us to raise the minimum libraw release we support?

Reply:
Not sure but I fixed the case at least in 6e0ff75f1fde987a3839ec3598684168e959cd2b to match what's in master